cmake/Modules/FindJson.cmake

```diff
@@ -12,7 +12,7 @@ include(LibFindMacros)
 
 set(Json_FIND_REQUIRED FALSE)
 set(Json_FIND_QUIETLY TRUE)
-libfind_pkg_detect(Json nlohmann_json FIND_PATH nlohmann/json.hpp)
+libfind_pkg_detect(Json nlohmann_json VERSION >= 3.6.0 FIND_PATH nlohmann/json.hpp)
```

Comment:
Won't work for ubuntu1804.
Either add the build steps in this module or add nlohman_json 3.6.0 as backport to your PPA